### PR TITLE
Branch to update to new dgl version

### PR DESCRIPTION
**Pull request by mikemhenry**
I am still building DGL packages, so right now only testing what I have, which is a python 3.9 build. We have some errors to fix to get it to work with `dgl 1.1.0`

### CODE REVIEW

**Comment by mikemhenry** [comment]
Okay sorry for the noise, the test failures now in CI are from API changes in DGL

**Comment by mikemhenry** [comment]
Sweet, CPU builds work!

**Comment by mikemhenry** [comment]
Some links to migrating the API
https://docs.dgl.ai/en/0.8.x/generated/dgl.function.copy_src.html
https://github.com/dmlc/dgl/blob/master/docs/migrate-guide-0.5.md

**Comment by mikemhenry** [comment]
OSX failures right now are from the osx builds missing from the label I am using, as soon as the ubuntu ones finish I can re-run the failed osx builds and it should be fine, but I don't want to interrupt the current linux builds

File: devtools/conda-envs/espaloma.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: espaloma-test
 channels:
   - conda-forge
-  - dglteam
+  - mmh/label/beta-post-patch
```
Comment (mikemhenry) [inline devtools/conda-envs/espaloma.yaml:4]:
```suggestion
```

**Review by ijpulidos — APPROVED** [review]
Looks good to me!

**Pull request merged**